src/saveload/saveload.h

```diff
@@ -381,9 +381,13 @@ enum SaveLoadVersion : uint16_t {
 
 	SLV_COMPANY_ALLOW_LIST,                 ///< 335  PR#12337 Saving of list of client keys that are allowed to join this company.
 	SLV_GROUP_NUMBERS,                      ///< 336  PR#12297 Add per-company group numbers.
+<<<<<<< HEAD
 	SLV_INCREASE_STATION_TYPE_FIELD_SIZE,   ///< 337  PR#12572 Increase size of StationType field in map array
 	SLV_ROAD_WAYPOINTS,                     ///< 338  PR#12572 Road waypoints
 
+	SLV_COMPANY_INAUGURATED_PERIOD,         ///< 339  PR#12798 Companies show the period inaugurated in wallclock mode.
+>>>>>>> 4a745b8737 (Codechage #12365|2402114: Moved Inaugurated Period saveload and changed comment to reflect the current master branch)
```

Comment:
This conflict wasn't quite resolved. It should look like this:

```suggestion
	SLV_INCREASE_STATION_TYPE_FIELD_SIZE,   ///< 337  PR#12572 Increase size of StationType field in map array
	SLV_ROAD_WAYPOINTS,                     ///< 338  PR#12572 Road waypoints
	SLV_COMPANY_INAUGURATED_PERIOD,         ///< 339  PR#12798 Companies show the period inaugurated in wallclock mode.
```